include/jakalib/math.h

```diff
@@ -0,0 +1,42 @@
+#ifndef JAKALIB_MATH_H
+#define JAKALIB_MATH_H
+
+#include <algorithm>
+#include <cmath>
+
+namespace jakalib {
+namespace {
```

Comment:
Anonymous namespaces in headers don't hide anything. Perhaps a "detail" namespace and/or an "impl" header would be better.